apps/settings/elements/call_cf_settings.html

```diff
@@ -1,16 +1,12 @@
 <element name="call-cfSettings" extends="section">
```

Comment:
I would like to make the html aligned to the folder name under `panels`, so let's rename the `name` and file name to `call-forwarding`. Please also update the change to `index.html`.